docker_build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+# Setup yarn and build qmk_configurator inside the docker container.
+
+set -e
+#set -x
+
+# Install node
+. /root/.nvm/nvm.sh
+nvm install 11.12
+
+# Get yarn installed correctly
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
+apt-get update
```

Comment:
this has to be in the dockerfile

Reply:
Why is that?

Reply:
Because those calls can be cached into a docker layer

Reply:
My understanding is that it's still cached into a docker layer, and that it's a good idea to collapse your Dockerfile into as few layers as needed. Is my understanding wrong?

Reply:
Yes but the whole layer will have the apt-get update and all the install of yarn and the project itself

Reply:
Yes, and that was my intent in arranging it this way. It bundles all the app dependencies together and ensures that stale dependencies don't break the app.

Reply:
After talking to yan a bit yesterday we're revising the direction and goal of this PR. I'm going to rework it so that API requests are always proxied through nginx. This eliminates the need to `yarn build` at startup time, so I'll be refactoring this PR in a way that makes this conversation moot.